test-data/unit/check-incremental.test

```diff
@@ -5277,3 +5277,23 @@ from other import C
 class C:
     pass
 [out]
+
+[case testConditionalExceptionAliasOverride]
+import a
+[file a.py]
+import lib
+x = 1
```

Comment:
Thanks! Can you also check that using `lib.Exception` works here?

Reply:
Do you mean in this file? I can add `try/except` here.

Reply:
Yes, in this file. Something like `try: ... except lib.Exception:`.